lib/lookup.json

```diff
@@ -327,22 +297,6 @@
     "skip": "win32",
     "maintainers": "ljharb"
   },
-  "mkdirp": {
-    "head": true,
-    "skip": ["win32", true],
-    "maintainers": "isaacs"
```

Comment:
is this only skipped on windows? if so, it seems valuable to keep it for linux/mac

Reply:
My understanding is that when: `true` it will be skipped on all platforms.